docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -81,11 +81,14 @@ Return the address of the pending owner that was initiated by [`transferOwnershi
 ### transferOwnership
 
 ```solidity
-function transferOwnership(address newOwner) public {
+function transferOwnership(address newOwner) public
 ```
 
 Initiate an ownership transfer by setting the `newOwner` as `pendingOwner`.
 
+Requirments:
+- `pendingOwner` cannot be `address(this)`.
```

Comment:
_Suggested changes_

The `newOwner` to be set as the `pendingOwner` cannot be `address(this)`